helpers/database-import-scripts/rnacentral/generate_rnacentral_json.py

```diff
@@ -449,14 +542,42 @@ def load_xml(sample_id, insdc_accession=None):
             print(r.text)
     else:
         logging.error('Could not retrieve xml for accession {}'.format(sample_id))
-        logging.error(r.text)
+        if r.status_code == 404:
+            logging.warning("Reason: sample is not in ENA")
+            if sample_id not in ERROR_404:
+                ERROR_404.append(sample_id)
+        else:
+            logging.error("Error in full: {}".format(r.text))
+            sys.exit("Program is exiting - unable to query ENA for sample {}. Error code {}".format(sample_id, 
+                                                                                                    r.status_code))
         return None
 
 
-@retry(tries=5, delay=10, backoff=1.5)
 def run_xml_request(xml_url):
```

Comment:
Why did you remove retry? It wasn't working properly, or just to remove the dependency?

Reply:
I changed it to a more manual approach because I couldn't get this to work for my purpose (tracking missing samples).